rs/execution_environment/tests/subnet_size_test.rs

```diff
@@ -751,6 +754,7 @@ fn get_cycles_account_manager_config(subnet_type: SubnetType) -> CyclesAccountMa
             // verified subnet is using.
             update_message_execution_fee: Cycles::new(5_000_000),
             ten_update_instructions_execution_fee: Cycles::new(10),
+            ten_update_instructions_execution_fee_wasm64: Cycles::new(20),
```

Comment:
Should we also do the same where we use a local variable for the old fee and set the new to be 2x that?

Reply:
yes, fixed, thanks.